builtin/game/chatcommands.lua

```diff
@@ -161,7 +161,7 @@ core.register_chatcommand("grant", {
 
 core.register_chatcommand("grantme", {
 	params = "<privilege> | all",
-	description = "Grant privileges to yourself",
+	description = "Grant privilege to yourself",
```

Comment:
Wasn't 'privileges' more accurate?

Reply:
For perfect accuracy it would need to be “privilege or privileges” but I decided this is a mouthful.

Reply:
Grant a privilege/s to yourself.

Reply:
One privilege would still fall under "privileges" where as multiple wouldn't fall under "privilege", so  it would be better to leave this one as is.

Reply:
you could use 'privilege(s)'

Reply:
What Ezhh wrote, it's better unchanged.

Reply:
Priv descriptions are currently inconsistent between “privilege” and “privileges”. Decide for one, please.

Reply:
I'd favour plural form where plural is possible.

Reply:
Yes plural.